src/main.cpp

```diff
@@ -112,6 +112,8 @@ void ctrpf::PatchProcess(ctrpf::FwkSettings &settings) {
         //TODO: find out how to make the tempo hooks JP-compatible
         Megamix::Hooks::TempoHooks();
     }
+
+    Megamix::Patches::PatchMuseumExtraRows();
```

Comment:
wrap this in an `if (region == Region::US)` for the time being

Reply:
ah forgot about that